utils/node_local_test.py

```diff
@@ -3812,7 +3799,7 @@ def _run_test(ptl=None, function=None, test_cb=None):
             # long-running tests which dominate the time, so whilst a higher value here would
             # work there's no benefit in rushing to finish the quicker tests.  The long-running
             # tests are started first.
-            while len(threads) > 5:
+            while len(threads) > 4:
```

Comment:
Should `4` be a global or class constant?

Reply:
or a command line variable?  I think here is as good as anywhere.